Preparedness/Preparedness/ViewModifiers/WithRoundedRect.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  WithRoundedRect.swift
+//  Preparedness
+//
+//  Created by Sebastian Wild on 10/25/20.
+//  Copyright © 2020 CCDC. All rights reserved.
+//
+
+import SwiftUI
+
+struct WithRoundedRect: ViewModifier {
```

Comment:
Interesting - is this the Standard Naming Convention?

Maybe this could be renamed to `RoundedRectModifier`?

Some examples I googled around
* [struct GreenButtonStyle: ViewModifier](https://programmingwithswift.com/viewmodifier-in-swiftui/)
* [struct Watermark: ViewModifier](https://www.hackingwithswift.com/books/ios-swiftui/custom-modifiers)

Reply:
Yeah I agree the name is not ideal, during class I was torn on it so named it whatever and moved on.
I like `RoundedRectModifier` - will make that change 👍